.travis.yml

```diff
@@ -21,11 +21,16 @@ matrix:
       - { python: "3.7", env: DJANGO=2.2 }
       - { python: "3.7", env: DJANGO=master }
 
-      - { python: "3.7", env: TOXENV=base }
-      - { python: "3.7", env: TOXENV=lint }
-      - { python: "3.7", env: TOXENV=docs }
+      - { python: "3.8", env: DJANGO=2.0 }
+      - { python: "3.8", env: DJANGO=2.1 }
+      - { python: "3.8", env: DJANGO=2.2 }
```

Comment:
Hi @Djailla. Django 2.x doesn't support Python 3.8, so it isn't necessary to add them to the build matrix.

Reply:
But will django 2.x support it ?

Reply:
Nope. Supported versions are locked upon release. They wouldn't add support in a patch.

Reply:
[Django documentation](https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django) makes it clear that it will not

Reply:
Done